File: 3 - Intro to Conditional Statements/Solution.java
import java.util.Scanner;

public class Solution {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int n = scan.nextInt();
        scan.close();

        if (n % 2 == 0) {
            if (n < 6) System.out.println("Not Weird");
            else if (n < 21) System.out.println("Weird");
            else System.out.println("Not Weird");
        } else System.out.println("Weird");
    }
}

File: 3 - Intro to Conditional Statements/Solution.go
package main

import "fmt"

func main() {
	var n int
	for {
		fmt.Scan(&n)
		if n >= 1 && n <= 100 {
			break
		}
	}

	if n%2 == 0 {
		if n >= 2 && n <= 5 {
			fmt.Println("Not Weird")
		} else if n >= 6 && n <= 20 {
			fmt.Println("Weird")
		} else if n > 20 {
			fmt.Println("Not Weird")
		}
	} else {
		fmt.Println("Weird")
	}
}
